build.gradle

```diff
@@ -77,6 +77,9 @@ configure(javaProjects) {
             // AssertJ
             dependency "org.assertj:assertj-core:${ext.versionOf('assertj')}"
 
+            // Completable-futures
+            dependency "com.spotify:completable-futures:${ext.versionOf('completable-futures')}"
```

Comment:
I wonder if we should shade the dependencies that are not exposed in our public API so that the applications that depend on Armeria have less version conflict issues. What do you think? (Not saying this should be done in this PR. I can send one later.)